addon/mixins/authenticated-route-mixin.js

```diff
@@ -78,7 +81,15 @@ export default Mixin.create({
       let authenticationRoute = this.get('authenticationRoute');
       assert('The route configured as Configuration.authenticationRoute cannot implement the AuthenticatedRouteMixin mixin as that leads to an infinite transitioning loop!', this.get('routeName') !== authenticationRoute);
 
-      if (!this.get('_isFastBoot')) {
+      if (this.get('_isFastBoot')) {
+        const fastboot = getOwner(this).lookup('service:fastboot');
+        const cookies = getOwner(this).lookup('service:cookies');
+
+        cookies.write('ember_simple_auth-redirectTarget', transition.intent.url, {
+          path: '/',
+          secure: fastboot.get('request.protocol') === 'https'
+        });
+      } else {
```

Comment:
Yeah, the test failures are unrelated to your changes - looks like travis had some problems for a few days now.

It seems like sth. went wrong when rebasing - your branch contains 2078be3336dcc64b7067d922233518e99e172de3 which has already been merged to master a few days ago…